### PR TITLE
Added distillation for quantization example on textual inversion.

### PR DESCRIPTION
**Pull request by XinyuYe-Intel**
Added a distillation for quantization example on textual inversion.

### CODE REVIEW

**Review by patrickvonplaten — COMMENTED** [review]
Thanks for the PR! Could we try to not add images though or upload them to the Hub instead and link from there? We're trying to keep the GitHub as small as possible :-)

**Comment by XinyuYe-Intel** [comment]
> Thanks for the PR! Could we try to not add images though or upload them to the Hub instead and link from there? We're trying to keep the GitHub as small as possible :-)

Sure, we are trying to upload the images to the Hub now.

**Comment by sayakpaul** [comment]
Would be nice to have some overview of how the distillation is carried out. Might be even nice to add this to the `train_text_to_image.py` as the next step.

File: examples/research_projects/intel_opts/textual_inversion_dfq/text2images.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import os
+import torch
+from diffusers import AutoencoderKL, PNDMScheduler, StableDiffusionPipeline, UNet2DConditionModel
+from diffusers.pipelines.stable_diffusion import StableDiffusionSafetyChecker
+from neural_compressor.utils.pytorch import load
+from PIL import Image
+from transformers import CLIPFeatureExtractor, CLIPTextModel, CLIPTokenizer
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Simple example of a training script.")
```
Comment (sayakpaul) [inline examples/research_projects/intel_opts/textual_inversion_dfq/text2images.py:13]: Needs to change.

**Comment by XinyuYe-Intel** [comment]
> Hi, thanks a lot for this.
> 
> Would be nice to have some overview of how the distillation is carried out and some stats on the memory saving obtained due to distillation and int8 quantization.

Of course, I'll add these later.

**Comment by XinyuYe-Intel** [comment]
> Thanks for the PR! Could we try to not add images though or upload them to the Hub instead and link from there? We're trying to keep the GitHub as small as possible :-)

Done.

**Comment by XinyuYe-Intel** [comment]
> Hi, thanks a lot for this.
> 
> Would be nice to have some overview of how the distillation is carried out and some stats on the memory saving obtained due to distillation and int8 quantization.

Done.

**Comment by patrickvonplaten** [comment]
Hey @XinyuYe-Intel,

Could you run `make style` here please?

**Comment by XinyuYe-Intel** [comment]
> Hey @XinyuYe-Intel,
> 
> Could you run `make style` here please?

Hi @patrickvonplaten, I've ran `make style` and looks like we are all set.

**Comment by patrickvonplaten** [comment]
Super, let's merge :-)

**Pull request merged**